### ISSUE TITLE
UEN Code

### ISSUE THREAD

**Issue by iangoh0414** [typeyourname/PayNow-QR-Javascript#1]
Hi,

I try using your code with mobile and it works. However, when i change to 'UEN' and use company UEN, it does not work. Are you able to test your code again?

**Comment by limexd-em** [comment]
Yeah I noticed that was a problem when I tried to use this code too.  Seems like the maintainer hasn't touched it for a while.

Did up another module specifically for use for UEN based QR code generation. Do check it out and let me know if it works.

https://github.com/ThunderQuoteTeam/PaynowQR

https://www.npmjs.com/package/paynowqr
